### PR TITLE
GG-7502: si-protected-user-list-admin-frontend: move entry ID from form to URL

### PR DESCRIPTION
**Pull request by jhkldev**

### CODE REVIEW

**Comment by jhkldev** [comment]
test this please

**Review by ModisR — CHANGES_REQUESTED** [review]

File: test/util/Generators.scala [hunk 1]
```diff
@@ -52,8 +52,9 @@ trait Generators {
 
   val sautrGen: Gen[SaUtr] = Gen.const(new SaUtrGenerator().nextSaUtr)
 
+  val entryIdGen: Gen[Option[String]] = Gen.some(nonEmptyStringGen)
```
[the remark this answers is not in the document]
Comment (jhkldev) [reply test/util/Generators.scala]: I've updated the PR to use `nonEmptyStringGen` directly.

**Comment by jhkldev** [comment]
test this please

**Review by ModisR — APPROVED** [review]
LGTM

**Pull request merged**